docs/source/usage/modes/search.rst

```diff
@@ -94,10 +94,7 @@ The following commands are available in search mode
 
 .. describe:: toggletags
 
-    flip presence of tags on this thread.
-    A tag is considered present if at least one message contained in this
-    thread is tagged with it. In that case this command will remove the tag
-    from every message in the thread.
+    flip presence of tags on this thread. A tag is considered present if at least one message contained in this thread is tagged with it. In that case this command will remove the tag from every message in the thread.
```

Comment:
So, the problem with that is it makes the docs in the python code and the rst not the same, which makes the docs tests fail. Maybe the right solution is to drop this change and find a better way to test the docs?